bin/v-add-mail-domain-webmail

```diff
@@ -13,38 +13,38 @@
 #----------------------------------------------------------#
 
 # Argument definition
-user=$1
-domain=$2
-webmail=$3
-restart="$4"
-quiet=$5
+user=${1}
+domain=${2}
+webmail=${3}
+restart=${4}
+quiet=${5}
```

Comment:
Honestly, I can't understand the "abuse" of {curly braces}...

Its use cases are usually handling arrays, default values of variables (${var:-} ${var:=} ...) etc.